features/zoek-met-burgerservicenummer/gba/actuele-reisdocumenten-gba.feature

```diff
@@ -0,0 +1,72 @@
+#language: nl
+
+@gba
+Functionaliteit: zoeken van de actuele reisdocumenten van een persoon
+
+  Rule: bij zoeken van reisdocumenten met burgerservicenummer worden alleen reisdocumenten geleverd die volgens de registratie nog in het bezit zijn van de persoon
+
+    Abstract Scenario: de persoon heeft een <omschrijving> en heeft geen ander reisdocument
+      Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+      | naam                                                                    | waarde       |
+      | soort reisdocument (35.10)                                              | PN           |
+      | nummer reisdocument (35.20)                                             | NE3663258    |
+      | datum uitgifte Nederlands reisdocument (35.30)                          | 20221106     |
+      | datum einde geldigheid reisdocument (35.50)                             | 20371106     |
+      | datum inhouding dan wel vermissing Nederlands reisdocument (35.60)      | 20221229     |
+      | aanduiding inhouding dan wel vermissing Nederlands reisdocument (35.60) | <aanduiding> |
+      Als gba reisdocumenten wordt gezocht met de volgende parameters
+      | naam                | waarde                     |
+      | type                | ZoekMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                  |
+      | fields              | reisdocumentnummer         |
+      Dan heeft de response 0 reisdocumenten
+
+      Voorbeelden:
+      | aanduiding | omschrijving                                                |
+      | I          | reisdocument ingeleverd                                     |
+      | R          | van rechtswege vervallen reisdocument                       |
+      | V          | vermist reisdocument                                        |
+      | .          | vermist of ingehouden reisdocument met onbekende aanduiding |
+
+    Scenario: de persoon heeft een reisdocument ingeleverd en heeft een ouder reisdocument van dezelfde soort nog in bezit
```

Comment:
Dit kan toch niet voorkomen? Het nieuwe reisdocument (dat nu wordt ingeleverd) kan alleen zijn uitgegeven als het oude is ingeleverd en/of als vermist etc. is opgegeven.